src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +27,10 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream().filter(word -> word.length() > 12)
+                .sorted((a, b) -> b.length() - a.length())
+                .distinct()
+                .forEach(word -> System.out.println(word.toLowerCase()));
```

Comment:
>stream을 사용할 때 가독성을 위한 개행 컨벤션이 있을까요?

명확히 정해진 컨벤션은 없지만 호출된 메서드를 정확히 구분해주기 위해 보통 한번의 호출이후에는 개행을 해주는 편입니다.  
```suggestion
        words.stream()
                .filter(word -> word.length() > 12)
                .sorted((a, b) -> b.length() - a.length())
                .distinct()
                .forEach(word -> System.out.println(word.toLowerCase()));
```